pom.xml

```diff
@@ -162,6 +162,7 @@
                     <failOnError>false</failOnError>
                     <failOnWarnings>false</failOnWarnings>
                     <source>8</source>
+                    <javadocExecutable>${java.home}/bin/javadoc/</javadocExecutable>
```

Comment:
This has to be removed, we have a secondary pom handling javadoc creation via the github actions workflow.

Reply:
Ah yes, I forgot about that line. I was having an issue with mavin not finding the javadoc command on my machine for some reason, so I had to add that line to get it to work properly. I set up my JAVA_HOME variable but it didn't seem to care. I'll remove that now.